### PR TITLE
make sure the type is lowercase

### PR DESCRIPTION
**Pull request by VladDubrovskis**
/cc @GlynnPhillips

### CODE REVIEW

**Comment by GlynnPhillips** [comment]
Small comment but looks good

**Comment by VladDubrovskis** [comment]
@GlynnPhillips what's the small comment? :)

File: server/middleware/display-variants.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
 'use strict';
 
 module.exports = (req, res, next) => {
-	let type = req.params.type;
+	let type = req.params.type ? req.params.type.toLowerCase() : false;
```
Comment (GlynnPhillips) [inline server/middleware/display-variants.js]: Can this false be replaced with `trial` and then remove the second condition on line 10?

**Comment by GlynnPhillips** [comment]
@VladDubrovskis https://github.com/Financial-Times/next-product-selector/pull/184/files#r79412521

**Comment by VladDubrovskis** [comment]
clearly was too eager for the comment 😂  will do the changes required

**Pull request merged**